tox.ini

```diff
@@ -12,5 +12,5 @@ deps =
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
     django111: Django>=1.11,<1.12
-    django20b1: Django>=2.0b1,<2.1
+    django20: Django>=2.0,<2.1
```

Comment:
this still needs to read `>=2.0b1` or pip won't find it

Reply:
Ok, I changed it...